.gitmodules

```diff
@@ -28,40 +28,6 @@
 [submodule "generators/block-inclusivecache-sifive"]
 	path = generators/sifive-cache
 	url = https://github.com/sifive/block-inclusivecache-sifive.git
-[submodule "toolchains/riscv-tools/riscv-gnu-toolchain"]
-	path = toolchains/riscv-tools/riscv-gnu-toolchain
-	url = https://github.com/riscv/riscv-gnu-toolchain.git
-[submodule "toolchains/riscv-tools/riscv-gnu-toolchain-prebuilt"]
-	path = toolchains/riscv-tools/riscv-gnu-toolchain-prebuilt
-	url = https://github.com/ucb-bar/chipyard-toolchain-prebuilt.git
-	shallow = true
-[submodule "toolchains/riscv-tools/riscv-isa-sim"]
-	path = toolchains/riscv-tools/riscv-isa-sim
-	url = https://github.com/riscv/riscv-isa-sim.git
-[submodule "toolchains/riscv-tools/riscv-pk"]
-	path = toolchains/riscv-tools/riscv-pk
-	url = https://github.com/riscv/riscv-pk.git
-[submodule "toolchains/riscv-tools/riscv-tests"]
-	path = toolchains/riscv-tools/riscv-tests
-	url = https://github.com/riscv/riscv-tests.git
-[submodule "toolchains/riscv-tools/riscv-openocd"]
-	path = toolchains/riscv-tools/riscv-openocd
-	url = https://github.com/riscv/riscv-openocd.git
-[submodule "toolchains/esp-tools/riscv-gnu-toolchain"]
-	path = toolchains/esp-tools/riscv-gnu-toolchain
-	url = https://github.com/ucb-bar/esp-gnu-toolchain.git
-[submodule "toolchains/esp-tools/riscv-isa-sim"]
-	path = toolchains/esp-tools/riscv-isa-sim
-	url = https://github.com/ucb-bar/esp-isa-sim.git
-[submodule "toolchains/esp-tools/riscv-pk"]
-	path = toolchains/esp-tools/riscv-pk
-	url = https://github.com/riscv/riscv-pk.git
-[submodule "toolchains/esp-tools/riscv-tests"]
-	path = toolchains/esp-tools/riscv-tests
-	url = https://github.com/ucb-bar/esp-tests.git
-[submodule "toolchains/libgloss"]
-	path = toolchains/libgloss
-	url = https://github.com/ucb-bar/libgloss-htif.git
 [submodule "vlsi/hammer"]
```

Comment:
Can we leave these submodules+build_toolchains.sh in? iI thought we are keeping the submodules, just removing them from the "default" instructions.

Reply:
I'll re-add the modules once they work (or some version of them). Right now, I was thinking of two approaches:

- Pointing to the conda feedstock repos (aka the conda build recipes). Those repos have the conda requirements listed out, a pointer to the src repos, and a script to build a conda package.
- Point to the src repos aka the new submodules that hold what was `toolchains/esp-tools` and `toolchains/riscv-tools`. Each of those src repos has its own `build-toolchains.sh` script.

Reply:
Re-added